backend/index.js

```diff
@@ -1,9 +1,9 @@
 const path = require('path');
```

Comment:
this whole file also should be moved to src/

Reply:
done